anaconda_anon_usage/utils.py

```diff
@@ -73,8 +77,10 @@ def _debug(s, *args, error=False):
 
 
 def _random_token(what="random"):
-    data = os.urandom(16)
-    result = base64.urlsafe_b64encode(data).strip(b"=").decode("ascii")
+    # base64 encoding captures 6 bits per character.
+    # Generate enough random bytes to ensure all charaaters are random
+    data = os.urandom((TOKEN_LENGTH * 6 - 1) // 8 + 1)
+    result = base64.urlsafe_b64encode(data).decode("ascii")[:TOKEN_LENGTH]
```

Comment:
That's an interesting change, with the token length of 22 we're bumping up the size of the random data to 17, can you elaborate why this is needed? Why limit the encoded result?

Reply:
Sure! I concede this is strictly not necessary to appear in this PR, it's a separate issue. It arose when I was considering the other variant of org tokens we were considering, embedding them in the client token itself.

In short, 22 characters of base64-encoded data can encode 6 * 22 = 132 bits = 16.5 bytes of information. If I only generate 16 random bytes, the base64 encoding still must generate 22 characters to contain it, but the last character is not completely random.

Reply:
I'm going to remove this change for now, as it is not central to the purpose of this PR

Reply:
Oh, I hadn't considered that, thanks for elaborating, a separate PR is a good idea, right